CMakeLists.txt

```diff
@@ -48,6 +48,7 @@ target_include_directories(azure_ulib_c
     PRIVATE
         ${PROJECT_SOURCE_DIR}/deps/umock-c/inc
         ${PROJECT_SOURCE_DIR}/deps/azure-macro-utils-c/inc
+        ${PROJECT_SOURCE_DIR}/deps/azure-core-c/inc
```

Comment:
this should not be needed. this should be taken care of with `target_link_libraries` below

Reply:
Interesting. 

---
In reply to: [564928784](https://github.com/Azure/azure-ulib-c/pull/57#discussion_r564928784) [](ancestors = 564928784)